### PR TITLE
Update Kubernetes Terminating Gateway Docs

### PR DESCRIPTION
**Pull request by t-eckert**
### Description

While working with terminating gateways, I noticed a few areas of improvement and spruced up the docs.

### Testing & Reproduction steps
* In the case of bugs, describe how to replicate
* If any manual tests were done, document the steps and the conditions to replicate
* Call out any important/ relevant unit tests, e2e tests or integration tests you have added or are adding

### Links
Include any links here that might be helpful for people reviewing your PR (Tickets, GH issues, API docs, external benchmarks, tools docs, etc). If there are none, feel free to delete this section.

Please be mindful not to leak any customer or confidential information. HashiCorp employees may want to use our internal URL shortener to obfuscate links.

### PR Checklist

* [ ] updated test coverage
* [x] external facing docs updated
* [x] not a security concern

### CODE REVIEW

**Review by david-yu — APPROVED** [review]
Wow thank you!

File: website/content/docs/k8s/connect/terminating-gateways.mdx [hunk 1]
```diff
@@ -306,7 +313,7 @@ $ kubectl apply --filename service-intentions.yaml
 
 ### Define the external services as upstreams for services in the mesh
 
-Finally define and deploy the external services as upstreams for the internal mesh services that wish to talk to them.
+As a final step, you may define and deploy the external services as upstreams for the internal mesh services that wish to talk to them.
```
Comment (mmisztal1980) [inline website/content/docs/k8s/connect/terminating-gateways.mdx:316]: Is this step applicable when transparent proxy is enabled?

File: website/content/docs/k8s/connect/terminating-gateways.mdx [hunk 2]
```diff
@@ -88,34 +94,35 @@ Registering the external services with Consul is a multi-step process:
...
 You can provide a `caFile` to secure traffic between unencrypted clients that connect to external services through the terminating gateway.
 Refer to [Create the configuration entry for the terminating gateway](#create-the-configuration-entry-for-the-terminating-gateway) for details.
 
-Also note that regardless of the `protocol` specified in the `ServiceDefaults`, [L7 intentions](/docs/connect/config-entries/service-intentions#permissions) are not currently supported with `ServiceDefaults` destinations.
+-> **Note:** Regardless of the `protocol` specified in the `ServiceDefaults`, [L7 intentions](/docs/connect/config-entries/service-intentions#permissions) are not currently supported with `ServiceDefaults` destinations.
 
 Create a `ServiceDefaults` custom resource for the external service:
 
-<CodeBlockConfig filename="serviceDefaults.yaml">
+<CodeBlockConfig filename="service-defaults.yaml">
```
Comment (mmisztal1980) [inline website/content/docs/k8s/connect/terminating-gateways.mdx:125]: Please provide information about k8s namespaces in context of Enterprise and OSS consul.

Comment (mmisztal1980) [inline website/content/docs/k8s/connect/terminating-gateways.mdx:125]: What if OSS Consul is running in the `consul` namespace and the workload is in a "dev" namespace? In which namespace  should the `ServiceDefaults` resource be placed?
- dev?
- consul?
- default?

Comment (mmisztal1980) [inline website/content/docs/k8s/connect/terminating-gateways.mdx:125]: Please provide information about k8s namespaces in context of Enterprise and OSS consul.

File: website/content/docs/k8s/connect/terminating-gateways.mdx (hunk 1 above)
Comment (mmisztal1980) [inline website/content/docs/k8s/connect/terminating-gateways.mdx:316]: Is this step applicable when transparent proxy is enabled?

**Review by mmisztal1980 — COMMENTED** [review]
The documentation fails to answer some basic questions. I've highlighted them in the PR.

File: website/content/docs/k8s/connect/terminating-gateways.mdx [hunk 3]
```diff
@@ -88,34 +94,35 @@ Registering the external services with Consul is a multi-step process:
 
 ### Register external services with Consul
 
-There are two ways to register an external service with Consul:
-1. If [`TransparentProxy`](/docs/connect/transparent-proxy) is enabled, the preferred method is to declare external endpoints in the [`destination`](/docs/connect/config-entries/service-defaults#terminating-gateway-destination) field of `ServiceDefaults`.
-1. You can add the service as a node in the Consul catalog.
+You may register an external service with Consul using `ServiceDefaults` if
```
Comment (mmisztal1980) [inline website/content/docs/k8s/connect/terminating-gateways.mdx:97]: What is the expectated outcome in the UI? Grerater than 0 amount of linked services in terminating gateway's UI?

**Pull request merged**